CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to the versioning scheme outlined in the [README.md](README.md).
 
+## [Unreleased]
+
+### Fixed
+
+- Handle the case where a bitcoin node returns zero headers (#3588)
```

Comment:
This won't be `[Unreleased]` once it's merged to master -- it should be `[2.1.0.0.1]`